Code-the-Dream-School/dd-prac-team1-front
# Techdebt/testing setup (completes PRA-202)

## One Line Description
Add a sample test and GHA workflow for running tests before merging in pull requests

## Requirements
- Tests should all pass now locally
- Tests should run as part of GitHub Actions for PRs
- Tests should pass in GHA


## Test Steps
- Run `npm test` on this branch
- When the terminal shows you the testing dialogue, type `a`
- Verify that the tests run and pass

## UI/UX Outcome
N/A
